### PR TITLE
[Samples] Updated the exception handling middleware sample to handle multiple output binding case

### PR DESCRIPTION
**Pull request by kshyju**
Updated the exception handling middleware sample to handle multiple output binding case.

### CODE REVIEW

File: samples/CustomMiddleware/CustomMiddleware.csproj [hunk 1]
```diff
@@ -13,9 +13,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.8.0-preview1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.9.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.5.0-preview1" OutputItemType="Analyzer" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Storage" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0" />
```
Comment (liliankasem) [inline samples/CustomMiddleware/CustomMiddleware.csproj:19]: Don't we want to keep `OutputItemType="Analyzer"`?

File: samples/CustomMiddleware/HttpTriggerWithMultipleOutputBindings.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            response.WriteString("Success!");
+
+            return new MyOutputType()
+            {
+                Name = "Azure Functions",
+                HttpResponse = response
+            };
+        }
+    }
+    public class MyOutputType
```
Comment (liliankasem) [inline samples/CustomMiddleware/HttpTriggerWithMultipleOutputBindings.cs:31]: nit: new line

**Review by liliankasem — APPROVED** [review]

File: samples/CustomMiddleware/CustomMiddleware.csproj (hunk 1 above)
Comment (kshyju) [reply samples/CustomMiddleware/CustomMiddleware.csproj:19]: Nope.

Comment (liliankasem) [reply samples/CustomMiddleware/CustomMiddleware.csproj:19]: Just for my understanding why? :D

Comment (kshyju) [reply samples/CustomMiddleware/CustomMiddleware.csproj:19]: It is needed only when you add as a project reference. For package reference, that is not needed.

Comment (liliankasem) [reply samples/CustomMiddleware/CustomMiddleware.csproj:19]: Good to know, we should probably remove this from templates next time we do an update

Comment (kshyju) [reply samples/CustomMiddleware/CustomMiddleware.csproj:19]: It is already removed from templates.

**Review by satvu — APPROVED** [review]
lgtm!

**Pull request merged**